src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java

```diff
@@ -368,6 +370,36 @@ public MultiverseWorld getFirstSpawnWorld() {
         return world;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void setJoinLocation(String destination) {
+        // Probably just mean they dont want join location, so we ignore.
+        if (destination == null || destination.length() == 0) {
+            this.joinLocation = null;
+            return;
+        }
+
+        this.joinLocation = this.plugin.getDestFactory().getDestination(destination);
+        if (this.joinLocation == null || this.joinLocation instanceof InvalidDestination) {
+            Logging.warning("Unable to set join location. Invalid join location: '" + plugin.getMVConfig().getJoinLocation() + "'!");
+            Logging.warning("See available mv destinations here: https://github.com/Multiverse/Multiverse-Core/wiki/Destinations");
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Location getJoinLocation(Player player) {
+        if (this.joinLocation == null || this.joinLocation instanceof InvalidDestination) {
+            Logging.warning("Unable get join location for " + player.getName() + " as join location set is invalid!");
```

Comment:
`...as join location set is invalid!` should be `...as the join location set is invalid!`.